.pipelines/submodules-pipeline.yaml

```diff
@@ -0,0 +1,319 @@
+              name: cni-dropgz-test
+            cns_linux_amd64:
+              arch: amd64
+              os: linux
+              name: cns
+            npm_linux_amd64:
+              arch: amd64
+              os: linux
+              name: npm
+        steps:
+          - template: containers/container-template.yaml
+            parameters:
+              name: $(name)
+              os: $(os)
+              arch: $(arch)
+      - job: containerize_linux_arm64
+        displayName: Build Images
+        variables:
+          TAG: $[ stagedependencies.setup.env.outputs['EnvironmentalVariables.Tag'] ]
+        pool:
+          name: "$(BUILD_POOL_NAME_LINUX_ARM64)"
+        strategy:
+          matrix:
+            acncli_linux_arm64:
+              arch: arm64
+              os: linux
+              name: acncli
+            cni_dropgz_test_linux_arm64:
+              arch: arm64
+              os: linux
+              name: cni-dropgz-test
+            cns_linux_arm64:
+              arch: arm64
+              os: linux
+              name: cns
+            npm_linux_arm64:
+              arch: arm64
+              os: linux
+              name: npm
+        steps:
+          - template: containers/container-template.yaml
+            parameters:
+              name: $(name)
+              os: $(os)
+              arch: $(arch)
+      - job: containerize_windows_amd64
+        timeoutInMinutes: 120
+        displayName: Build Images
+        variables:
+          TAG: $[ stagedependencies.setup.env.outputs['EnvironmentalVariables.Tag'] ]
+        pool:
+          name: "$(BUILD_POOL_NAME_WINDOWS_AMD64)"
+        strategy:
+          matrix:
+            cns_windows_amd64:
+              arch: amd64
+              name: cns
+            npm_windows_amd64:
+              arch: amd64
+              name: npm
+        steps:
+          - template: containers/container-template-windows.yaml
+            parameters:
+              name: $(name)
+              arch: $(arch)
+              tag: $(TAG)
+
+  - stage: publish
+    displayName: Publish Multiarch Manifests
+    dependsOn:
+      - containerize
+    jobs:
+      - job: manifest
+        displayName: Compile Manifests
+        pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT)"
+        strategy:
+          matrix:
+            acncli:
+              name: acncli 
+              platforms: linux/amd64 linux/arm64
+            cni_dropgz_test:
+              name: cni-dropgz-test
+              platforms: linux/amd64 linux/arm64
+            cns:
+              name: cns
+              platforms: linux/amd64 linux/arm64 windows/amd64
+            npm:
+              name: npm
+              platforms: linux/amd64 linux/arm64 windows/amd64
+        steps:
+          - template: containers/manifest-template.yaml
+            parameters:
+              name: $(name)
+              platforms: $(platforms)
+  
+  # - template: singletenancy/ipam-dropgz/ipam-dropgz-cilium-e2e-job-template.yaml
```

Comment:
I see this template commented out, should it be enabled? I couldn't find any other reference to ipam-dropgz-cilium-e2e-job or the swift one

Reply:
I commented while I was testing a new parameter with the original swifte2e and ciliume2e template. I am planning to remove the ipam-dropgz templates

Reply:
I'm restructuring things based off of the feedback I've gotten so far